File: array_sum.py
#calculate and return sum of all elements of a list
def sumList(p,sum = 0):
    le = len(p)
    if le==0:
        return sum
    else:
        sum = sum+p[le-1]
        p.pop()
        return sumList(p,sum)
    

l = eval(input("Enter the list: "))
p = sumList(l)
print(p)

File: Java/ArraySum.java
public class ArraySum {

    static int arr[] = { 11, 2, 3, 4, 8 };

    static int Sum(int A[], int n) {
        
        if (n <= 0)
            return 0;
        return (Sum(A, n - 1) + A[n - 1]);
    }

    public static void main(String[] args){
      
        System.out.println("The sum of elements of the array is: "+ Sum(arr, arr.length));
    }
}